packages/cli/src/util/runner.ts

```diff
@@ -0,0 +1,38 @@
+import { getHttpOperationsFromResource } from '@stoplight/prism-http';
+import { IPrismHttpServer } from '@stoplight/prism-http-server/src/types';
+import * as chokidar from 'chokidar';
+import { CreatePrismOptions } from './createServer';
+
+export function runPrism(createPrism: Function, options: CreatePrismOptions, spec: string) {
```

Comment:
This function has some inconsistencies. Sometimes you use `async/await` and sometimes you go with the `Promises`. I do not mind which one you prefer, but pick one and keep using it for the whole flow.